src/AppBundle/Controller/PostController.php

```diff
@@ -30,4 +30,16 @@ public function onlPostViewAction($postId)
     {
 
     }
+
+    public function testJenkinsCI()
+    {
+        $string = 'string';
+
+        if ($string){
```

Comment:
**Reporter**: CHECKSTYLE
**Rule**: PEAR.ControlStructures.ControlSignature.Found
**Severity**: ERROR
**File**: /var/lib/jenkins/workspace/test2/src/AppBundle/Controller/PostController.php L38

Expected &quot;if (...) {\n&quot;; found &quot;if (...){\n&quot;


*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1080081996>*